### PR TITLE
Preparing the code to being published on npm

### PR DESCRIPTION
**Pull request by giovannism20**
- changed the title of the package on readme
- added license (agpl 3.0)
- make some updates on package json

after this would be great to create an account for librecode on npm and publish it - create some tests would be great also!

i've used this tutorial to make it: https://github.com/DIGOARTHUR/how-to-build-NPM-package

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,9 @@
 {
-  "name": "vue-pdf-editor",
+  "name": "pdf-editor-librecode",
```
Comment (vitormattos) [inline package.json:2]:
```suggestion
  "name": "pdf-editor-libresign",
```

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# vue-pdf-editor
+# pdf-editor-librecode
```
Comment (vitormattos) [inline README.md:1]:
```suggestion
# pdf-editor-libresign
```

File: package.json [hunk 3]
```diff
@@ -40,5 +42,19 @@
...
     "not ie 11"
-  ]
+  ],
+  "license": "AGPL-3.0",
+  "bugs": {
+    "url": "https://github.com/LibreSign/pdf-editor/issues"
+  },
+  "homepage": "https://github.com/LibreSign/pdf-editor#readme",
+  "keywords": [
+    "LibreCode",
```
Comment (vitormattos) [inline package.json:52]:
```suggestion
    "LibreSign",
    "VueJS",
    "signature",
    "signing",
    "sign",
```

**Review by vitormattos — APPROVED** [review]

**Comment by giovannism20** [comment]
can u provide it @vitormattos ? "after this would be great to create an account for librecode on npm and publish it - create some tests would be great also!"

**Comment by vitormattos** [comment]
I also created this: https://www.npmjs.com/org/libresign

**Comment by giovannism20** [comment]
thank u @vitormattos

**Comment by giovannism20** [comment]
i'm making the merge to start tests

**Comment by vitormattos** [comment]
@giovannism20 @enieber started to fix the problems at this project to work fine like the original project. Don't go ahead with other tasks besides this PR at this moment.

**Pull request merged**